unskript-ctl/unskript_ctl_factory.py

```diff
@@ -134,17 +134,26 @@ def update_credentials_to_uglobal(self):
                 creds_json_files.extend(glob.glob(pattern, recursive=True))
                 break
         self.creds_json_files = creds_json_files
+        c_data = {}
         for creds_json_file in creds_json_files:
             with open(creds_json_file, 'r', encoding='utf-8') as f:
-                c_data = json.load(f)
-                if c_data.get('metadata').get('connectorData') == '{}':
-                    continue 
-                mapping[c_data.get('metadata').get('type')] = {"name": c_data.get('metadata').get('name'), 
+                try:
+                    c_data = json.load(f)
+                except:
+                    # If creds file is empty, continue to the next creds file
+                    pass 
+                finally:
+                    if c_data.get('metadata').get('connectorData') == '{}':
+                        continue 
+                    mapping[c_data.get('metadata').get('type')] = {"name": c_data.get('metadata').get('name'), 
                                                             "id": c_data.get('id')}
         self.uglobals['default_credentials'] = mapping
 
     def _banner(self, msg: str):
         print('\033[4m\x1B[1;20;42m' + msg + '\x1B[0m\033[0m')
+
+    def _highlight(self, msg: str):
+        print('\033[4m\x1B[1;20;40m' + msg + '\x1B[0m\033[0m')
```

Comment:
Lets not use this kind of encoding, there is already TBL_HDR_* defined in utils, use that to print

Reply:
Done.